### PR TITLE
Allow an empty value for dhcp options to set the default options. Cor…

### PR DESCRIPTION
**Pull request by krjackso**
…rectly diff default dhcp options with local config

### CODE REVIEW

File: lib/vpc/models/DhcpConfig.rb [hunk 1]
```diff
@@ -68,6 +68,7 @@ def populate!(aws)
       def diff(aws)
         diffs = []
 
+        aws_domain_name_servers = (aws.domain_name_servers || []).sort
         if @domain_name_servers.sort != aws.domain_name_servers.sort
```
Comment (msiebert) [inline lib/vpc/models/DhcpConfig.rb]: did you mean
`if @domain_name_servers.sort != aws_domain_name_servers`

File: lib/vpc/models/VpcDiff.rb [hunk 2]
```diff
@@ -56,8 +56,16 @@ def self.subnets(aws, local)
       end
 
       def self.dhcp(aws, local)
-        dhcp_diffs = local.diff(aws)
-        if !dhcp_diffs.empty?
+        puts "aws #{aws} local #{local}"
```
Comment (msiebert) [inline lib/vpc/models/VpcDiff.rb]: is this debug?

File: lib/vpc/models/DhcpConfig.rb (hunk 1 above)
Comment (krjackso) [inline lib/vpc/models/DhcpConfig.rb]: Yup. Was trying to catch a NilClass exception... and then left it in 2 more times

File: lib/vpc/models/VpcDiff.rb (hunk 2 above)
Comment (krjackso) [inline lib/vpc/models/VpcDiff.rb]: So embarassing ....

**Pull request merged**